crypto_verify.js

```diff
@@ -0,0 +1,34 @@
+const assert = require('nanoassert')
+
+module.exports = {
+  crypto_verify_16,
+  crypto_verify_32,
+  sodium_memcmp,
+  sodium_is_zero
+}
+
+function vn (x, xi, y, yi, n) {
+  var i, d = 0
+  for (i = 0; i < n; i++) d |= x[xi + i] ^ y[yi + i]
+  return (1 & ((d - 1) >>> 8)) - 1
+}
+
+function crypto_verify_16(x, xi, y, yi) {
+  return vn(x, xi, y, yi, 16)
+}
+
+function crypto_verify_32(x, xi, y, yi) {
+  return vn(x, xi, y, yi, 32)
+}
+
+function sodium_memcmp (a, b) {
+  assert(a.byteLength = b.byteLength, 'buffers must be the same size')
```

Comment:
I am just a casual observer of this PR as I am excited about this! but it looks like the equality comparison is missing another `=` character here, unless I am missing something

Reply:
thanks! fixed in 56416de